app/lib/pre_assembly.rb

```diff
@@ -0,0 +1,12 @@
+require 'fileutils'
+require 'erb'
+
+
+
+# auto require any project specific files
+Dir[File.dirname(__FILE__) + '/pre_assembly/project/*.rb'].each { |file| require "pre_assembly/project/#{File.basename(file)}" }
```

Comment:
If Rails autoloading is actually working, this line should be unnecessary.  If it **is** necessary, it means we have some further filename/classname alignment to do.